conf/binac.config

```diff
@@ -0,0 +1,25 @@
+/*
+vim: syntax=groovy
+-*- mode: groovy;-*-
+ * ----------------------------------------------------------------------------
+ *  Nextflow config file for use with Singularity on BINAC cluster in Tuebingen
+ * ----------------------------------------------------------------------------
+ * Defines basic usage limits and singularity image id.
+ */
+
+singularity {
+    enabled = true
+}
+
+process {
+    beforeScript = 'module load devel/singularity/2.4.1'
+    executor = 'pbs'
+    container = 'docker://$wf_container'
```

Comment:
Hmm, you may need double quotes here. Also we may need to use `{}` to defer execution? Difficult to actually test this until we have more tagged releases available!

Reply:
Hi @apeltzer - does this work as it stands? Let me know if you'd like me to merge anyway.

Reply:
Oh, sorry. I corrected this in NGI-RNAseq, but forgot to do so here. I'll change it and send again.

Reply:
There you go!